Sources/Core/CelyConstants.swift

```diff
@@ -77,3 +58,8 @@ public extension UIWindow {
         }
     }
 }
+
+public enum AccessibilityOptions {
```

Comment:
This name is confusing, because these options don't seem to have anything to do with Accessibility in the sense of providing access to people with disabilities. Would something along the lines of `AccessControlOptions` be more in line?

Reply:
Great suggestion!

I believe this is a much better name.